tests/testthat/test-theme_elements_gtsummary.R

```diff
@@ -0,0 +1,212 @@
+# Package-wide Unit Tests-------------------------------------------------------
+## pkgwide-fn:prependpvalue_fun-------------------------------------------------
+test_that("pkgwide-fn:prependpvalue_fun works", {
+
+  # Create a theme#
+  my_theme_1 <-
+    list(
+      # Prepend p value, with 3 place digits
+      "pkgwide-fn:prependpvalue_fun" = label_style_pvalue(digits = 3, prepend_p = TRUE)
+    )
+
+  # Set the theme
+  set_gtsummary_theme(my_theme_1)
```

Comment:
Instead of setting a theme in semi-open code, let's use `with_gtsummary_theme()`, because it's well-confined to the operations that are called within it.